python_modules/dagster/dagster/core/definitions/pipeline.py

```diff
@@ -1,19 +1,16 @@
 import six
 
-from toposort import toposort_flatten
-
 from dagster import check
 
+from ..utils import toposort_flatten
```

Comment:
i generally think we should avoid relative imports like this. i find they break in unreliable ways

Reply:
fine by me